metadata.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: synapse
+display-name: Synapse
+summary: Matrix synapse chat server.
+description: |
+  A charm for the matrix synapse chat server.
+  Synapse is a drop in replacement for other chat servers like Mattermost and Slack.
+  This charm is useful if you want to spin up your own chat instance.
+docs: https://discourse.charmhub.io/t/synapse-documentation-overview/11358
+issues: https://github.com/canonical/synapse-operator/issues
+maintainers:
+  - https://launchpad.net/~canonical-is-devops
+source: https://github.com/canonical/synapse-operator
+assumes:
+  - k8s-api
+
+containers:
+  synapse:
+    resource: synapse-image
+    mounts:
+      - storage: data
+        location: /data
+  synapse-nginx:
+    resource: synapse-nginx-image
+
+resources:
+  synapse-image:
+    type: oci-image
+    description: OCI image for Synapse
+  synapse-nginx-image:
+    type: oci-image
+    description: OCI image for Synapse NGINX
+
+storage:
+  data:
+    type: filesystem
+    location: /data
+
+provides:
+  grafana-dashboard:
+    interface: grafana_dashboard
+  metrics-endpoint:
+    interface: prometheus_scrape
+
+requires:
+    ingress:
+        interface: ingress
+        limit: 2
```

Comment:
What is the reason for making this 2 please? I would have guessed that 1 ingress is sufficient? Perhaps worthwhile documenting the reason since others in the future might also wonder

Reply:
We should probably consider dropping this relation

Reply:
Related: #49